corehq/util/es/interface.py

```diff
@@ -107,17 +101,11 @@ def _fix_hits_in_results(self, results):
 
 
 class ElasticsearchInterface1(AbstractElasticsearchInterface):
-    _disallowed_index_settings = (
-        'max_result_window',
-    )
+    pass
 
 
 class ElasticsearchInterface2(AbstractElasticsearchInterface):
-    _disallowed_index_settings = (
-        'merge.policy.merge_factor',
-        'store.throttle.max_bytes_per_sec',
-        'store.throttle.type',
-    )
+    pass
 
 
 ElasticsearchInterface = {
```

Comment:
All of this can be removed now, and `AbstractElasticsearchInterface` can be renamed to `ElasticsearchInterface`

Reply:
The ES7 interface is different, so we will need `AbstractElasticsearchInterface` See https://github.com/dimagi/commcare-hq/pull/28270/files#diff-35449daeec5614e63f3a53d79459660aR128

I have removed ES2 interface though.